lib/line-builders.js

```diff
@@ -0,0 +1,99 @@
+'use strict'
+
+const {
+  prettifyErrorLog,
+  prettifyObject
+} = require('./utils')
+
+const lineBuilders = [
+  (lineParts, { prettified, opts }) => {
```

Comment:
I'm not fond of this big array of anonymous functions. As written, it is not clear what each function is meant to do. Even so much as a proper function declaration with a name can make things much clearer. Even better would be a jsdoc block preceding each function. Look at `lib/utils.js` for a nice example.

Reply:
That's a good point. I just pushed a pretty big update that significantly improves this by using functionally cohesive classes with parse and build methods that make them far more readable and maintainable.